2018/5xxx/CVE-2018-5560.json

```diff
@@ -1,18 +1,101 @@
 {
-   "CVE_data_meta" : {
-      "ASSIGNER" : "cve@mitre.org",
-      "ID" : "CVE-2018-5560",
-      "STATE" : "RESERVED"
+   "CVE_data_meta": {
+      "ASSIGNER": "cve@rapid7.com",
+      "DATE_PUBLIC": "2018-12-17T14:05:00.000Z",
```

Comment:
Is 12/17 correct or should it be 12/27 when we published our blog post?

Reply:
Whoops, should be Dec 27. Fixing!